R/correctionsataglance-data.R

```diff
@@ -117,12 +117,23 @@ formatDateRange <- function(startD, endD){
 }
 
 formatDataList <- function(dataIn, type, ...){
+  
```

Comment:
I'm not sure, exactly, this isn't code that I wrote. If you think there could be improvements, perhaps we should create some jiras for future work?